src/Sarif/SplittingStrategy.cs

```diff
@@ -6,42 +6,43 @@ namespace Microsoft.CodeAnalysis.Sarif
     /// <summary>
     /// Enumerates the grouping strategies provided out of the box by the WorkItemFiler.
```

Comment:
>grouping [](start = 23, length = 8)

"grouping" => "splitting"

Reply:
Yes. Also updated type name reference to 'SarifWorkItemFiler'

---
In reply to: [381007170](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381007170) [](ancestors = 381007170)